src/components/Project/ProjectPage.tsx

```diff
@@ -1,15 +1,73 @@
 "use client";
 
-import { useState } from "react";
+import { useCallback, useEffect, useState } from "react";
 import ProjectTabs from "../ProjectTabs";
-import { ProjectTab } from "@/types/const";
+import { ProjectIDType, ProjectTab } from "@/types/const";
 import ProjectDetailsComponent from "../ProjectMetaDataTable/ProjectDetailsComponent";
 import ContractDetailsPage from "../ProjectContract/ContractDetailsPage";
 import ProjectDistribution from "./ProjectDistribution";
 import MockData from "./project.json";
+import { useAccount } from "wagmi";
+import getDecodedAttestationData from "@/lib/eas/getDecodedAttestationData";
+import { useProjectProvider } from "@/context/ProjectProvider";
+import { useParams } from "next/navigation";
+import { usePaymasterProvider } from "@/context/Paymasters";
+import { getProjects } from "@/lib/eas/getProjects";
 
 const ProjectPage = () => {
   const [tabContent, setTabContent] = useState<ProjectTab>("project");
+  const [mockData, setMockData] = useState(null);
```

Comment:
is this still mockData?
- looks like results are coming from an API request right?

Reply:
if data is not mocked, please refactor this variable name

Reply:
thanks for the additional details.


could you please refactor this variable to remove the `mock` prefix?
```suggestion
const ProjectPage = () => {
  const [tabContent, setTabContent] = useState<ProjectTab>("project");
  const [projectData, setProjectData] = useState(null);
```